docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+
+version: "3.0"
+services:
+  rails:
+    build: .
+    command: bundle exec foreman start -f Procfile_development
+    depends_on:
+      - postgres
+      - redis
+    environment:
+      REDIS_URL: redis://redis:6379
+      PGHOST: postgres
+      PGUSER: postgres
+    env_file:
+    - docker.env
```

Comment:
I did not want to duplicate the .env file, but I found out that the format: `VARIABLE: value` is not valid for docker, needs to be `VARIABLE=value` see: https://docs.docker.com/compose/env-file/

And maintaining two files almost identical is pretty hard. I'll look further for another approach, or if anyone has a suggestion.

Reply:
Hey Bruno! Switching the dotenv file to use equals (which works fine for the app).

I'll merge #1564 as soon as it passes tests

Thanks for taking a look at this 😀

Reply:
merged @wwwbruno 🙂

Reply:
Thank you for the fast response @sethherr